ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh

```diff
@@ -21,15 +32,23 @@ finished()
   set +o pipefail
   set +o errexit
 
-  echo "Fetching kubeconfig, other credentials..."
-  scp "${SSHOPTS[@]}" "root@${IP}:/root/dev-scripts/ocp/*/auth/kubeconfig" "${SHARED_DIR}/"
-  scp "${SSHOPTS[@]}" "root@${IP}:/root/dev-scripts/ocp/*/auth/kubeadmin-password" "${SHARED_DIR}/"
+  # Injects additional dev-scripts env vars from a multistage step
```

Comment:
We should avoid as much as possible to inject specif configuration code (especially for agent) in the generic steps. I'd prefer that the step (in dev-scripts) would create dummy kube* files (and hopefully in future we could also close this gap when the e2e workflow will be completed)